narwhals/series.py

```diff
@@ -519,6 +519,40 @@ def mean(self) -> Any:
         """
         return self._compliant_series.mean()
 
+    def skew(self) -> Any:
+        """
+        Calculate the sample skewness of the Series.
+
+        Returns:
+            The sample skewness of the Series.
+
+        Examples:
+            >>> import pandas as pd
+            >>> import polars as pl
+            >>> import narwhals as nw
+            >>> s = [1, 2, 3, 4, 5]
```

Comment:
Can we use the non trivial example of the expr case?

Reply:
Sure, adjusted this now